docs/src/developer/style.md

```diff
@@ -12,12 +11,9 @@
 [Julia style guide](https://docs.julialang.org/en/v1/manual/style-guide/index.html) as reference.
 * Read [Julia contribution guidelines](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#general-formatting-guidelines-for-julia-code-contributions), notably its line length limit.
 * Read [Julia guidelines for docstrings](https://docs.julialang.org/en/v1/manual/documentation/index.html).
-* Read [JuMP coding standards](http://www.juliaopt.org/JuMP.jl/dev/style). In particular, note its policies on
-  * [whitespace](http://www.juliaopt.org/JuMP.jl/dev/style/#Whitespace-1)
-  * [return statements](http://www.juliaopt.org/JuMP.jl/dev/style/#Return-statements-1)
-  * [variable names](http://www.juliaopt.org/JuMP.jl/dev/style/#Use-of-underscores-within-names-1).
-* Read [The Zen of Python](https://www.python.org/dev/peps/pep-0020).
+* Read [BlueStyle](https://github.com/invenia/BlueStyle/) style guide.
 * Consider using a plugin that configures your text editor to use [EditorConfig](https://editorconfig.org/) settings.
+* Consider using [JuliaFormatter.jl](https://github.com/domluna/JuliaFormatter.jl).
```

Comment:
We could save ourselves a lot of time if this was required.  I can imagine a CI test that runs this on changed files and fails if it changes any text.  Is there any reason not to do this?